documentation/models.py

```diff
@@ -306,6 +306,12 @@ def __unicode__(self):
     def overridden(self):
         return False
 
+    def get_children(self):
+        return Map.objects.filter(parent=self).all()
```

Comment:
great solution! I think it would be helpful to add a comment here explaining that we're making sure to return a `Map` instead of a `Page` so that `get_absolute_url` can be overridden.